ts-e2e/package.json

```diff
@@ -14,22 +14,22 @@
   "dependencies": {
     "@azure/event-hubs": "^1.0.8",
     "@types/chai": "^3.5.1",
-    "@types/debug": "4.1.2",
+    "@types/debug": "^4.1.1",
     "@types/mocha": "^2.2.41",
-    "@types/node": "^7.0.8",
-    "@types/uuid": "^2.0.29",
+    "@types/node": "^9.3.0",
+    "@types/uuid": "^3.4.3",
     "azure-iot-common": "^1.9.0-preview.0",
     "azure-iot-device": "^1.9.0-preview.0",
     "azure-iot-device-amqp": "^1.9.0-preview.0",
     "azure-iot-device-http": "^1.9.0-preview.0",
     "azure-iot-device-mqtt": "^1.9.0-preview.0",
     "azure-iothub": "^1.9.0-preview.0",
-    "chai": "^3.5.0",
+    "chai": "^4.2.0",
     "debug": "^4.1.1",
     "mocha": "^5.2.0",
-    "tslint": "^5.11.0",
+    "tslint": "^5.9.1",
     "typescript": "^3.1.6",
```

Comment:
>typescript": "^3.1.6" [](start = 5, length = 21)

Danger - 3.1.5

Reply:
again, pierre's project.  I assume he wanted typescript 3.x for a reason.

---
In reply to: [270532046](https://github.com/Azure/azure-iot-sdk-node/pull/521#discussion_r270532046) [](ancestors = 270532046)